Dockerfile

```diff
@@ -4,10 +4,7 @@ MAINTAINER Team Aruha, team-aruha@zalando.de
 
 WORKDIR /
 ADD build/libs/nakadi.jar nakadi.jar
-ADD plugins plugins/
-ADD scm-source.json /scm-source.json
 
 EXPOSE 8080
 
-# run the server when a container based on this image is being run
-ENTRYPOINT java $JAVA_OPTS -Dloader.path=/plugins -Djava.security.egd=file:/dev/./urandom -jar nakadi.jar
\ No newline at end of file
+ENTRYPOINT java -jar nakadi.jar
```

Comment:
I checked it now. Currently I don't experience any problems with that. So I think we can remove that 
(if the problem happens again in the future we can return it back)